packages/swapkit/cli/src/screens/Exit.tsx

```diff
@@ -0,0 +1,15 @@
+import { useApp } from 'ink';
+import React, { useEffect } from 'react';
+
+const Exit = () => {
+  const app = useApp();
+
+  useEffect(() => {
+    app.exit();
+    process.exit();
+  }, [app]);
+
+  return <></>;

```

Comment:
```suggestion
  return null;
```

seems more reasonable